populous/compat.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+from builtins import range
+from past.builtins import basestring
+
+PY2 = sys.version_info[0] == 2
+
+
+try:
+    from itertools import izip
+except ImportError:
+    izip = zip
```

Comment:
same remark than for `range` I prefer `zip = izip`. But I'm pretty sure futures already has that.